overview/default.nix

```diff
@@ -194,7 +158,7 @@ let
               lib.concatMapAttrsStringSep "\n" (
                 name: val:
                 optionalString (val.module != null) (
-                  render.options.many [ type name ] (
+                  render.options [ type name ] (
```

Comment:
We had a draft commit from a pairing session where we extracted these mysterious lists into named values for readability, can we have that back please?